src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -112,34 +112,27 @@ class VisEditorVisualization extends Component {
         };
       });
     }
-  }
+  };
 
   hasShowPanelIntervalValue() {
     const type = get(this.props, 'model.type', '');
+    const interval = get(this.props, 'model.interval') || AUTO_INTERVAL;
```

Comment:
`get(this.props, 'model.interval', AUTO_INTERVAL)`

Reply:
I want to set 'auto' value only if model.interval is not defined. In your example it will replaced in cases of empty string or '0'. I prefer to save my variant